### DISCUSSION TITLE
[nodriver] elem.click() does not work on droplist element HELP!

### DISCUSSION THREAD

**Discussion by vedmed123** [ultrafunkamsterdam/undetected-chromedriver#2098] [Q&A]
elem.click() does not work on droplist element on page.
Link to page: https://www.kz.amway.com/uhazhivayushchij-tonalnyj-krem-artistry-future-glow/p/126718
Droplist element with colors.
Element locate by text "dropdown-select". Click on this element, but list not drop down.
![2024-12-01_22-51-49](https://github.com/user-attachments/assets/7c4d3c17-4026-4cb2-9daf-abc334e5d445)
![2024-12-01_22-51-01](https://github.com/user-attachments/assets/154c6a6b-139e-4203-af60-5b5e0efbc289)

**Comment by WhiskeyHammer** [comment]
I noticed this too. Using the mouse_click method worked for me.
